files/start-strongswan.sh

```diff
@@ -1,20 +1,59 @@
 #!/bin/sh
 set -eo pipefail
 
+_initialize() {
+    echo "Start: run initializations."
+    _create_vti
+    echo "End: run initializations."
+}
+
+_create_vti() {
+
+# set charon.install_virtual_ip = no to prevent the daemon from also installing the VIP
+
+    if [ -n "$IPSEC_VTI_KEY" ]
+    echo "IPSEC_VTI_KEY set, creating VTI interface."
+    then
+        set -o nounset
+        set -e
+        set -x
```

Comment:
please remove -x or set it just for some DEBUG mode.

Reply:
done